<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>center4</class>
 <widget class="QWidget" name="center4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>27</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*#center4 {
	background-color: transparent;
    border-image: url(&quot;:/icons/duba.png&quot;);
}

QLabel {
	color: rgb(255, 255, 255);
	font: 8pt &quot;Microsoft YaHei UI&quot;;
}</string>
  </property>
  <widget class="QLabel" name="cpuUse">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>7</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="netUp">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>7</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="netDown">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>7</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="memUse">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>7</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
